relayer/scripts/run_relayer.sh

```diff
@@ -42,6 +42,13 @@ if [[ -n "${SIGNER_CID}" ]]; then
   ARGS+=(--signer-cid=${SIGNER_CID})
 fi
 
+if [[ -n "${PAYOUT_ADDRESS}" ]]; then
+  ARGS+=(--payout-address=${PAYOUT_ADDRESS})
+else
+  echo "Payout address required but not provided." 1>&2
+  exit 1
```

Comment:
Why do we treat lack of `PAYOUT_ADDRESS` as error condition here?

Reply:
We shouldn't be doing this - I missed it. Fixed now.